.gitignore

```diff
@@ -87,8 +87,9 @@ report.asciidoc
 /bazel
 /bazel-*
 .bazelrc.user
+.bazelrc.cache
```

Comment:
we may have to be on the lookout for an unintentional git add here - e.g switching  to an older branch after a newer bootstrap.